File: Java/프로그래머스/0/120813. 짝수는 싫어요/짝수는 싫어요.java
class Solution {
    public int[] solution(int n) {
        int count = 0;
        int num=0;
        for(int i=1;i<=n;i++){
            if(i%2!=0) count++;
        }
        int answer[]=new int[count];
        for(int i=1; i*2-1<=n ; i++){
            answer[num]=i*2-1;
            num++;
        }
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120813. 짝수는 싫어요/짝수는 싫어요.js
function solution(n) {
    var count=0;
    var m=0;
    for(var i=1; i<=n; i++){
        if(i%2!=0) {
            count++;
        }
    }
    var answer = count;
    var answer = new Array(count);
    for(var i=1; i<=n ; i++){
        if(i%2!=0) {
            answer[m]=i;
            m++;
        }
    }
    return answer;
}